server/api/video.api.js

```diff
@@ -17,22 +17,35 @@ const headersValidation = Joi.object({
   authorization: Joi.string().required(),
 }).options({ allowUnknown: true });
 
-module.exports = function(server, mongoose, logger) {
+module.exports = function (server, mongoose, logger) {
   // Update Video Segments Endpoint
-  (function() {
+  (function () {
     const Log = logger.bind(Chalk.magenta('Update Video Segments'));
 
     Log.note('Generating Update Video Segments Endpoint');
 
-    const updateVideoSegmentsHandler = async function(request, h) {
+    const updateVideoSegmentsHandler = async function (request, h) {
```

Comment:
Just a general note. Formatting changes are good, however I've found that PRs are much easier to review when the format changes are kept separate from the actual code/logic changes, as it is much clearer what is being changed. I usually leave format changes to separate PRs to help prevent confusion.

Reply:
We will also need to at some point make sure we are using the same styling/formatting tools. Not a huge priority though.

Reply:
Justin, I believe it was one of the linting tools built into Webstorm that made some formatting changes.  I noticed it had split some lines, and in a couple cases even added parentheses in calls.  Once I noticed this, I disabled this linting going forward.